### PR TITLE
refactor: source flags trigger to include machine type

### PR DESCRIPTION
**Pull request by sshanzel**
Should be good to review. The only difference here between each function is the removal of the `AND type = 'squad'`.

Note: it is a good sign that some tests failed, it means it worked. Will update those cases in a few seconds.

### CODE REVIEW

File: src/migration/1717589112159-SourceStatsTrigger.ts
Comment (sshanzel) [inline src/migration/1717589112159-SourceStatsTrigger.ts:81]: For the rollback, we just need to copy the initial function:
https://github.com/dailydotdev/daily-api/blob/722a3016a868efeda40cd1b0629a825483ff3823/src/migration/1714633817175-SquadTotalPostsTrigger.ts#L7-L21

File: src/migration/1717589112159-SourceStatsTrigger.ts
Comment (sshanzel) [inline src/migration/1717589112159-SourceStatsTrigger.ts:106]: For the rollback, we just need to copy the initial function:

https://github.com/dailydotdev/daily-api/blob/722a3016a868efeda40cd1b0629a825483ff3823/src/migration/1714662136871-SquadTotalUpvotesTrigger.ts#L9-L32

File: src/migration/1717589112159-SourceStatsTrigger.ts
Comment (sshanzel) [inline src/migration/1717589112159-SourceStatsTrigger.ts:126]: For the rollback, we just need to copy the initial function:

https://github.com/dailydotdev/daily-api/blob/722a3016a868efeda40cd1b0629a825483ff3823/src/migration/1714667947050-SquadTotalViewsTrigger.ts#L7-L25

File: src/migration/1717589112159-SourceStatsTrigger.ts
Comment (rebelchris) [inline src/migration/1717589112159-SourceStatsTrigger.ts:8]: Not sure if maybe we should delete and re-add with better naming?
`increment_source_posts_count` (but only if it doesn't add much overhead)

**Review by rebelchris — APPROVED** [review]
One minor suggestion.
So we just need to be careful to quickly run the bin script once we merge right?
Else the trigger will already fire before hand

**Comment by sshanzel** [comment]
> So we just need to be careful to quickly run the bin script once we merge right?

Just as long as we run the script after merging, we should be good whichever time we pick since they will just be back to what they should really count.

**Comment by rebelchris** [comment]
@sshanzel Oh right I guess it will accumulate whatever on that time 👍

**Review by rebelchris — APPROVED** [review]

**Review by nensidosari — APPROVED** [review]

**Comment by sshanzel** [comment]
Ran through the migrations locally again, and applied to the local DB. After refresh, there are no more triggers specific to squads as shown below.

Screenshot 2024-06-06 at 4 37 04 PM

**Pull request merged**
